src/routes/(proposals)/proposal/[slug]/proposal-timeline.svelte

```diff
@@ -0,0 +1,119 @@
+<script lang="ts">
+  import type { ProposalPackagePricingStoryblok } from '$types/bloks';
+  import clsx from 'clsx';
+
+  export let data;
+  export let type: string;
+  export let pricing: ProposalPackagePricingStoryblok[] | undefined;
+
+  const { deliverables, rows, totalDays, totalMonths } = data;
+
+  let projectManagement = pricing?.find(
+    (resource) => resource.department.name === 'Project Manager'
+  );
+  let qualityAssurance = pricing?.find(
+    (resource) => resource.department.name === 'Quality Assurance'
+  );
+</script>
+
+<div class="container mx-auto 2xl:px-12 mt-10 md:mt-14 lg:mt-20">
+  <div
+    class={clsx(
+      'relative overflow-x-scroll',
+      'grid gap-1 grid-cols-[max-content,_max-content] py-3'
+    )}
+  >
+    <div class="col-start-1 row-start-1"></div>
+    <div class="col-start-2">
+      <div
+        class="grid gap-1 px-1 auto-cols-[20px] grid-rows-2 border-b shadow-sm"
+        style="background: linear-gradient(to right, #DDD 1px, transparent 1px); background-size: 120px;"
```

Comment:
This has an hardcoded color, make sure it's handled for both light and dark mode